production/loki-mixin/dashboards/dashboard-loki-operational.json

```diff
@@ -2543,9 +2543,9 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=\"$cluster\", namespace=\"$namespace\", pod=~\"ingester.*\"}",
+              "expr": "sum by(pod) (rate(container_cpu_usage_seconds_total{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"ingester\"}[$__rate_interval]))",
               "intervalFactor": 3,
-              "legendFormat": "{{pod}}-{{container}}",
```

Comment:
Nit: if it ain't broke, rather don't fix it

Reply:
It is kinda borken now: `<pod>-<container>` now reads as `<ingester-zone-a-1>- <ingester>`

Reply:
Keep in mind not everyone uses zone-aware ingesters
That being said, I get now why you were making the change so I'm not overly worried

Reply:
yes i appreciate that people have many different settings. 
`container` doesn't provide more info. Probably `node` does but then it's a very long string, i didn't want to add. `pod` name gives enough to find the node if needed.